core/AssetManager/UIAssetFetcher/PluginUmdAssetFetcher.php

```diff
@@ -276,7 +276,16 @@ protected function getPriorityOrder()
     private static function getRelativePluginDirectory($plugin)
     {
         $result = self::getPluginDirectory($plugin);
-        $result = str_replace(PIWIK_INCLUDE_PATH . '/', '', $result);
+
+        $matomoPath = rtrim(PIWIK_INCLUDE_PATH, '/') . '/';
+        $webroots = array_merge(Manager::getAlternativeWebRootDirectories(), [$matomoPath]);
+
+        foreach ($webroots as $webroot) {
```

Comment:
It should be, since its coming as a key value pair
`foreach ($webroots as $webroot => $v) {`

Reply:
@AltamashShaikh modified